src/range_proof.rs

```diff
@@ -83,7 +98,14 @@ pub struct RangeProof {
 /// for aggregation_size in proof_batch {
 ///     // 1. Generators
 ///     let extension_degree = ExtensionDegree::Zero;
-///     let generators = RangeParameters::init(bit_length, aggregation_size, extension_degree).unwrap();
+///     let pc_gens = ristretto::create_pedersen_gens_with_extension_degree(extension_degree);
+///     let generators = RangeParameters::init(bit_length, aggregation_size, pc_gens).unwrap();
+///     let generators = RangeParameters::init(
+///         bit_length,
+///         aggregation_size,
+///         ristretto::create_degree_zero_pedersen_gens(),
+///     )
+///     .unwrap();
```

Comment:
```suggestion
```
This is a duplicate line - the former has clearer intent as `extension_degree` is re-used further down.